muted-tests.yml

```diff
@@ -251,8 +251,6 @@ tests:
 - class: org.elasticsearch.datastreams.DataStreamsClientYamlTestSuiteIT
   method: test {p0=data_stream/140_data_stream_aliases/Create data stream aliases using wildcard expression}
   issue: https://github.com/elastic/elasticsearch/issues/120890
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  issue: https://github.com/elastic/elasticsearch/issues/120816
```

Comment:
I wonder if you can still use the `muted-tests` infra by adding the line:
```
method: test {p0=ml/inference_crud*
```

It just seems simpler to try and keep all test muting done in the same place. Mixing muting methods seems like a recipe to forget that things are actually muted.

Reply:
> I wonder if you can still use the muted-tests infra by adding the line:

Well yes you can. I did not think to that and it's much cleaner. Thanks @benwtrent